packages/atlas/src/components/_inputs/Select/Select.tsx

```diff
@@ -119,23 +129,28 @@ export const _Select = <T extends unknown>(
           </ValueAndPlaceholderText>
           <SelectChevronWrapper>{isOpen ? <SvgActionChevronT /> : <SvgActionChevronB />}</SelectChevronWrapper>
         </SelectButton>
-        <SelectMenu {...getMenuProps()}>
-          {isOpen && (
-            <StyledList
-              scrollable
-              items={items
-                .filter((item) => !item.hideInMenu)
-                .map((item, index) => ({
-                  selected: item.value === selectedItemValue,
-                  highlight: highlightedIndex === index,
-                  label: item.name,
-                  ...item,
-                  ...getItemProps({ item: item.value, index, onClick: item.onClick, disabled: item.isSeparator }),
-                }))}
-              size={size === 'medium' ? 'small' : 'medium'}
-            />
-          )}
-        </SelectMenu>
+        {ReactDOM.createPortal(
+          <div ref={setDropdownRef} style={{ ...styles.popper }} {...attributes.popper}>
+            <SelectMenu {...getMenuProps()}>
+              {isOpen && (
+                <List
+                  scrollable
+                  items={items
+                    .filter((item) => !item.hideInMenu)
+                    .map((item, index) => ({
+                      selected: item.value === selectedItemValue,
+                      highlight: highlightedIndex === index,
+                      label: item.name,
+                      ...item,
+                      ...getItemProps({ item: item.value, index, onClick: item.onClick, disabled: item.isSeparator }),
+                    }))}
+                  size={size === 'medium' ? 'small' : 'medium'}
+                />
+              )}
+            </SelectMenu>
+          </div>,
+          document.body
+        )}
```

Comment:
I've noticed a bug. It happens on the video workspace. 

https://user-images.githubusercontent.com/51168865/207274405-8c632260-f885-4766-b6fd-6611c06ecb9c.mov

For some reason, it starts working when I'm scrolling.

Reply:
Also, sometimes the menu is misplaced: 
![image](https://user-images.githubusercontent.com/51168865/207275047-bb076ea7-38b8-4c48-b345-eed33cd6bed3.png)